src/error.rs

```diff
@@ -1073,6 +1083,16 @@ but it cannot be found.",
                     label,
                 };
 
+                write_project(buffer, diagnostic);
+            }
+            Error::Format {
+                input: FormatInput::STDIN,
+            } => {
+                let diagnostic = ProjectErrorDiagnostic {
+                    title: "STDIN has not been formatted".to_string(),
```

Comment:
I had a look around and it seems the canonical name is stdin (lowercase), at least according to GNU, Microsoft and the C++ reference.

Could we make this and the constructor lowercase? Thanks